--- C:\Users\Annie\RE-SEARCH\model.py
+++ C:\Users\Annie\RE-SEARCH\model.py
@@ -1,6 +1,6 @@
 class cnn2(nn.Module):
 
-    def __init__(self, num_class, batch_norm =True, dropout = 0.17, n1 = 5, n2 = 8, k1 = 3, k2 = 5, l1= 40, type = 'buttons'):
+    def __init__(self, num_class, batch_norm =False, dropout = 0.0, n1 = 5, n2 = 8, k1 = 3, k2 = 5, l1= 40, type = 'nobutton'):
         super(cnn2,self).__init__()
         self.type = type
         if type == 'buttons':
@@ -31,8 +31,7 @@
             self.bn2 = nn.BatchNorm1d(num_class)
 
 
-    def forward(self, x, batch_norm=False, loss_fnc = 'MSE',type='buttons'):
-        self.type=type
+    def forward(self, x, batch_norm=False, loss_fnc = 'MSE'):
         if (batch_norm):
             x = self.pool(F.relu(self.conv_bn1(self.dropout2(self.conv1(x)))))
             x = self.pool(F.relu(self.conv_bn2(self.dropout2(self.conv2(x)))))
@@ -44,8 +43,6 @@
                 x = F.softmax(x, dim=1)
             elif loss_fnc == 'MSE' and self.type == 'buttons':
                 x = F.sigmoid(x)
-            if self.type == 'buttons':
-                x = x.squeeze()
         else:
             x = self.pool(F.relu(self.dropout2(self.conv1(x))))
             x = self.pool(F.relu(self.dropout2(self.conv2(x))))
@@ -56,8 +53,6 @@
                 x = F.softmax(x, dim=1)
             elif loss_fnc == 'MSE' and self.type == 'buttons':
                 x = F.sigmoid(x)
-            if self.type == 'buttons':
-                x = x.squeeze()
 
         return x
 